roles/lib_openshift/src/lib/base.py

```diff
@@ -464,13 +464,12 @@ def add_custom_versions(versions):
     @staticmethod
     def openshift_installed():
         ''' check if openshift is installed '''
-        import yum
+        import rpm
```

Comment:
rpm libs aren't available? Atomic still has RPM and yum even if they're only informational...